registry/handlers/api_test.go

```diff
@@ -520,7 +520,11 @@ func doPushLayer(t *testing.T, ub *v2.URLBuilder, name string, dgst digest.Diges
 
 // pushLayer pushes the layer content returning the url on success.
 func pushLayer(t *testing.T, ub *v2.URLBuilder, name string, dgst digest.Digest, uploadURLBase string, body io.Reader) string {
-	resp, err := doPushLayer(t, ub, name, dgst, uploadURLBase, body)
+	buf := new(bytes.Buffer)
```

Comment:
I could also just change doPushLayer to accept a body of []byte rather than an io.Reader and it would be a much smaller change

@stevvooe would that work for you?